### ISSUE TITLE
On Windows Foreground goes behind video

### ISSUE THREAD

**Issue by gertfj** [molamil-com/codethewave-framework#1]
http://codepen.io/ramiroespada/pen/mAwXAJ

This looks fine on mac, but on Windows (Martin Dyrby's computer) the blue color renders behind the video mask

**Migrated comments:**

Ramiro Espada
@Nikolaj Stausbøl but the video plays masked inline?

Nikolaj Stausbøl
@Ramiro Espada Yes he can use the video and mask it normally. It's just anything he puts on the top layer goes behind the mask

Ramiro Espada
Wow, that's pretty odd but I'll try to fix it, thanks for posting!

**Comment by respada** [comment]
@gertfj Do you know in which version of IE we are experiencing this issue?

**Comment by gertfj** [comment]
No, I just moved it from Podio. It was reported by @staus so please try and ask him.
